### ISSUE TITLE
Relative lib returning incorrect values

### ISSUE THREAD

**Issue by tivac** [tivac/modular-css#298]
When `process.cwd()` doesn't match `cwd` bad things can happen in `lib/relative.js`.

```
process.cwd(): C:\ArenaNet\Source\BuildDevWebgw\Portal\Web\Products\Js2GuildWars2
cwd: Z:\Source\BuildDevWebgw\Portal\Web\Products\Js2GuildWars2
file: Public/preload/shared/video/index.css
out: C:/ArenaNet/Source/BuildDevWebgw/Portal/Web/Products/Js2GuildWars2/Public/preload/shared/video/index.css
```

if the `file` param is already absolute though then the call to `path.relative()` does the right thing.

```
process.cwd(): C:\ArenaNet\Source\BuildDevWebgw\Portal\Web\Products\Js2GuildWars2
cwd: Z:\Source\BuildDevWebgw\Portal\Web\Products\Js2GuildWars2
file: Z:\Source\BuildDevWebgw\Portal\Web\Products\Js2GuildWars2\Public\preload\pof\index.css
out: Public/preload/pof/index.css
```

I think this is fixable by adding a `path.isAbsolute(file)` check and only running `path.relative()` if `file` is an absolute path.

**Comment by tivac** [comment]
I changed `lib/relative.js`'s default export to this and it worked fine in the build that was failing.

```js
module.exports = function(cwd, file) {
    if(path.isAbsolute(file)) {
        file = path.relative(cwd, file);
    }

    return file.replace(sepRegex, "/");
};
```

**Comment by tivac** [comment]
Fixed by #300, hopefully.
